voluncheer/profiles/urls.py

```diff
@@ -2,15 +2,16 @@
 from django.urls import path
 
 from profiles.views.home import home
-from profiles.views.profile import ProfileView
+from profiles.views.profile import *
 
 urlpatterns = [
     path("", home, name="home"),
-    path("profile/", ProfileView.as_view(), name="profile"),
+    path(r'^profile/$', ProfileView.as_view(), name="profile"),
```

Comment:
Please keep the quote we are using the same unless it would require you to escape a quote within a string. We have been using the double-quote. Here and elsewhere.